core/storage/user/gae_models.py

```diff
@@ -272,9 +272,10 @@ class UserSubscriptionsModel(base_models.BaseModel):
     activity_ids = ndb.StringProperty(repeated=True, indexed=True)
     # IDs of collections that this user subscribes to.
     collection_ids = ndb.StringProperty(repeated=True, indexed=True)
-    # IDs of feedback thread ids that this user subscribes to.
+    # IDs of feedback thread ids that this user subscribes to
+    # (Deprecated, do not use) use general_feedback_thread_ids instead.
```

Comment:
Replace this whole comment (both lines) with "DEPRECATED. DO NOT USE. Use general_feedback_thread_ids instead."